app/assets/stylesheets/application.css.scss

```diff
@@ -49,3 +49,23 @@ header {
 .col-lg-10 {
     margin-left: 20px;
 }
+
+/* COMMENTS */ 
+.user_avatar {
+  float: left;
+  margin-right: 10px;
+  height: 50px;
+  width: 50px;
+}
```

Comment:
Could we add some empty lines between these styles? Makes it a bit easier to read as well.